core/templates/includes/tabs.html

```diff
@@ -1,45 +0,0 @@
-{% load custom_filters %}
```

Comment:
To be clear, Jessica said you could only because they are destroyed in the master branch currently :)